### PR TITLE
SEOImageUpdates: User Story 1463880, Small PR 2

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Making a smaller PR.

Fixes #1463880

Contributes to #10839

### CODE REVIEW

File: docs/framework/windows-workflow-foundation/how-to-create-a-state-machine-workflow.md [hunk 1]
```diff
@@ -173,7 +173,7 @@ Workflows can be constructed from built-in activities as well as from custom act
   
      The following example illustrates the completed workflow.  
   
-     ![Completed State Machine Workflow](./media/wfstatemachinegettingstartedtutorialcomplete.JPG "WFStateMachineGettingStartedTutorialComplete")  
+     ![Completed state machine workflow from the Getting Started Tutorial](./media/how-to-create-a-state-machine-workflow/getting-started-tutorial-complete-state-machine-workflow.jpg)
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/how-to-create-a-state-machine-workflow.md]:
```suggestion
     ![Illustration that shows the completed state machine workflow.](./media/how-to-create-a-state-machine-workflow/complete-state-machine-workflow.jpg)  
```

File: docs/framework/windows-workflow-foundation/state-machine-workflows.md [hunk 2]
```diff
@@ -17,14 +17,14 @@ A state machine is a well-known paradigm for developing programs. The <xref:Syst
   
  To create a state machine workflow, states are added to a <xref:System.Activities.Statements.StateMachine> activity, and transitions are used control the flow between states. The following screenshot, from the [Getting Started Tutorial](getting-started-tutorial.md) step [How to: Create a State Machine Workflow](how-to-create-a-state-machine-workflow.md), shows a state machine workflow with three states and three transitions. **Initialize Target** is the initial state and represents the first state in the workflow. This is designated by the line leading to it from the **Start** node. The final state in the workflow is named **FinalState**, and represents the point at which the workflow is completed.  
   
- ![Completed State Machine Workflow](./media/wfstatemachinegettingstartedtutorialcomplete.JPG "WFStateMachineGettingStartedTutorialComplete")  
+ ![Completed state machine workflow from the Getting Started Tutorial](./media/state-machine-workflows/getting-started-tutorial-complete-state-machine-workflow.jpg)
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/state-machine-workflows.md]:
```suggestion
 ![Illustration that shows the completed state machine workflow.](./media/state-machine-workflows/complete-state-machine-workflow.jpg)  
```

File: docs/framework/windows-workflow-foundation/state-machine-workflows.md [hunk 3]
```diff
@@ -17,14 +17,14 @@ A state machine is a well-known paradigm for developing programs. The <xref:Syst
...
+ ![Completed state machine workflow from the Getting Started Tutorial](./media/state-machine-workflows/getting-started-tutorial-complete-state-machine-workflow.jpg)  
   
  A state machine workflow must have one and only one initial state, and at least one final state. Each state that is not a final state must have at least one transition. The following sections cover creating and configuring states and transitions.  
   
 ## Creating and Configuring States  
  A <xref:System.Activities.Statements.State> represents a state in which a state machine can be in. To add a <xref:System.Activities.Statements.State> to a workflow, drag the **State** activity designer from the **State Machine** section of the **Toolbox** and drop it onto a <xref:System.Activities.Statements.StateMachine> activity on the [!INCLUDE[wfd1](../../../includes/wfd1-md.md)] surface.  
   
- ![WF4 State Machine Activities](./media/netframework4platformupdate1statemachineactivities.jpg "NETFramework4PlatformUpdate1StateMachineActivities")  
+ ![WF4 State Machine Activities](./media/state-machine-workflows/net-framework-platform-update-state-machine-activities.jpg)
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/state-machine-workflows.md]:
```suggestion
 ![Screenshot of the State Machine section of the Toolbox](./media/state-machine-workflows/state-machine-section-toolbox.jpg)  
```

File: docs/framework/windows-workflow-foundation/wf-features-in-the-rehosted-workflow-designer.md [hunk 4]
```diff
@@ -31,25 +31,25 @@ Windows Workflow Foundation (WF) in [!INCLUDE[net_v45](../../../includes/net-v45
...
- ![Variable and Argument Designer Context Menu](./media/designercontextmenu.png "DesignerContextMenu")
+ ![Variable and Argument Designer Context Menu](./media/wf-features-in-the-rehosted-workflow-designer/designer-context-menu.png)
 
 ### Auto-surround with Sequence
  Since a workflow or certain container activities (such as <xref:System.Activities.Statements.NoPersistScope>) can only contain a single body activity, adding a second activity required the developer to delete the first activity, add a <xref:System.Activities.Statements.Sequence> activity, and then add both activities to the sequence activity. Starting with [!INCLUDE[net_v45](../../../includes/net-v45-md.md)], when adding a second activity to the designer surface, a `Sequence` activity will be automatically created to wrap both activities. This feature is supported in the rehosted designer.
 
  The following screenshot shows a `WriteLine` activity in the `Body` of a `NoPersistScope`.
 
- ![Auto&#45;surround drop location](./media/autosurround1.png "AutoSurround1")
+ ![Auto-surround with WriteLine activity](./media/wf-features-in-the-rehosted-workflow-designer/auto-surround-write-line-activity.png)
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/wf-features-in-the-rehosted-workflow-designer.md]:
```suggestion
 ![A WriteLine activity in the Body of a NoPersistScope activity.](./media/wf-features-in-the-rehosted-workflow-designer/auto-surround-write-line-activity.png)
```

File: docs/framework/windows-workflow-foundation/wf-features-in-the-rehosted-workflow-designer.md [hunk 5]
```diff
@@ -31,25 +31,25 @@ Windows Workflow Foundation (WF) in [!INCLUDE[net_v45](../../../includes/net-v45
...
 
  The following screenshot shows the automatically created `Sequence` activity in the `Body` when a second `WriteLine` is dropped below the first.
 
- ![Automatically created sequence activity](./media/autosurround2.png "AutoSurround2")
+ ![Auto-surround with Sequence activity](./media/wf-features-in-the-rehosted-workflow-designer/auto-surround-sequence-activity.png)
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/wf-features-in-the-rehosted-workflow-designer.md]:
```suggestion
 ![An automatically created Sequence in the Body of a NoPersistScope.](./media/wf-features-in-the-rehosted-workflow-designer/auto-surround-sequence-activity.png)
```

File: docs/framework/windows-workflow-foundation/wf-features-in-the-rehosted-workflow-designer.md [hunk 6]
```diff
@@ -31,25 +31,25 @@ Windows Workflow Foundation (WF) in [!INCLUDE[net_v45](../../../includes/net-v45
...
 
 ### Pan Mode
  To more easily navigate a large workflow in the designer, pan mode can be enabled, allowing the developer to click and drag to move the visible portion of the workflow, rather than needing to use the scroll bars. The button to activate pan mode is in the lower right corner of the designer. This feature is supported in the rehosted designer.
 
  The following screenshot shows the pan button which is located at the bottom right corner of the workflow designer.
 
- ![Pan button in workflow designer](./media/panbutton.png "PanButton")
+ ![Use the pan button to move the visible portion of the workflow.](./media/wf-features-in-the-rehosted-workflow-designer/pan-button-visible-workflow.png)
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/wf-features-in-the-rehosted-workflow-designer.md]:
```suggestion
 ![The pan button highlighted in the workflow designer.](./media/wf-features-in-the-rehosted-workflow-designer/pan-button-workflow-designer.png)
```

File: docs/framework/windows-workflow-foundation/wf-features-in-the-rehosted-workflow-designer.md [hunk 7]
```diff
@@ -63,7 +63,7 @@ Windows Workflow Foundation (WF) in [!INCLUDE[net_v45](../../../includes/net-v45
 
  The following screenshot of the completed workflow from the [Getting Started Tutorial](getting-started-tutorial.md) shows the outline view with a sequential workflow.
 
- ![Outline View in Workflow Designer](./media/outlineviewinworkflowdesigner.jpg "OutlineViewinWorkflowDesigner")
+ ![Outline View with a sequential workflow](./media/wf-features-in-the-rehosted-workflow-designer/outline-view-in-workflow-designer.jpg)
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/wf-features-in-the-rehosted-workflow-designer.md]:
```suggestion
 ![Screenshot of outline view with a sequential workflow in Visual Studio](./media/wf-features-in-the-rehosted-workflow-designer/outline-view-in-workflow-designer.jpg)
```

File: docs/framework/windows-workflow-foundation/wf-features-in-the-rehosted-workflow-designer.md [hunk 8]
```diff
@@ -128,7 +128,7 @@ Windows Workflow Foundation (WF) in [!INCLUDE[net_v45](../../../includes/net-v45
   
  The following screenshot shows the completed state machine workflow from the [Getting Started Tutorial](getting-started-tutorial.md) step [How to: Create a State Machine Workflow](how-to-create-a-state-machine-workflow.md).  
   
- ![Completed State Machine Workflow](./media/wfstatemachinegettingstartedtutorialcomplete.JPG "WFStateMachineGettingStartedTutorialComplete")  
+ ![Completed state machine workflow from the Getting Started Tutorial](./media/wf-features-in-the-rehosted-workflow-designer/getting-started-tutorial-complete-state-machine-workflow.jpg)
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/wf-features-in-the-rehosted-workflow-designer.md]:
```suggestion
 ![Illustration that shows the completed state machine workflow.](./media/wf-features-in-the-rehosted-workflow-designer/complete-state-machine-workflow.jpg)  
```

File: docs/framework/windows-workflow-foundation/whats-new-in-wf-in-dotnet.md [hunk 9]
```diff
@@ -113,27 +113,27 @@ In [!INCLUDE[netfx40_short](../../../includes/netfx40-short-md.md)], variables a
...
+![Variable and Argument Designer Context Menu](./media/whats-new-in-wf-in-dotnet/designer-context-menu.png)
 
 ### <a name="BKMK_AutoSurround"></a> Auto-surround with Sequence
 
 Since a workflow or certain container activities (such as <xref:System.Activities.Statements.NoPersistScope>) can only contain a single body activity, adding a second activity required the developer to delete the first activity, add a <xref:System.Activities.Statements.Sequence> activity, and then add both activities to the sequence activity. Starting with [!INCLUDE[net_v45](../../../includes/net-v45-md.md)], when adding a second activity to the designer surface, a `Sequence` activity will be automatically created to wrap both activities.
 
 The following screenshot shows a `WriteLine` activity in the `Body` of a `NoPersistScope`.
 
-![Auto&#45;surround drop location](./media/autosurround1.png "AutoSurround1")
+![Auto-surround with WriteLine activity](./media/whats-new-in-wf-in-dotnet/auto-surround-write-line-activity.png)
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/whats-new-in-wf-in-dotnet.md]:
```suggestion
![A WriteLine activity in the Body of a NoPersistScope activity.](./media/whats-new-in-wf-in-dotnet/auto-surround-write-line-activity.png)
```

File: docs/framework/windows-workflow-foundation/whats-new-in-wf-in-dotnet.md [hunk 10]
```diff
@@ -113,27 +113,27 @@ In [!INCLUDE[netfx40_short](../../../includes/netfx40-short-md.md)], variables a
...
 
 The following screenshot shows the automatically created `Sequence` activity in the `Body` when a second `WriteLine` is dropped below the first.
 
-![Automatically created sequence activity](./media/autosurround2.png "AutoSurround2")
+![Auto-surround with Sequence activity](./media/whats-new-in-wf-in-dotnet/auto-surround-sequence-activity.png)
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/whats-new-in-wf-in-dotnet.md]:
```suggestion
![An automatically created Sequence in the Body of a NoPersistScope.](./media/whats-new-in-wf-in-dotnet/auto-surround-sequence-activity.png)
```

File: docs/framework/windows-workflow-foundation/whats-new-in-wf-in-dotnet.md [hunk 11]
```diff
@@ -113,27 +113,27 @@ In [!INCLUDE[netfx40_short](../../../includes/netfx40-short-md.md)], variables a
...
 
 ### <a name="BKMK_PanMode"></a> Pan Mode
 
 To more easily navigate a large workflow in the designer, pan mode can be enabled, allowing the developer to click and drag to move the visible portion of the workflow, rather than needing to use the scroll bars. The button to activate pan mode is in the lower right corner of the designer.
 
 The following screenshot shows the pan button which is located at the bottom right corner of the workflow designer.
 
-![Pan button in workflow designer](./media/panbutton.png "PanButton")
+![Use the pan button to move the visible portion of the workflow.](./media/whats-new-in-wf-in-dotnet/pan-button-visible-workflow.png)
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/whats-new-in-wf-in-dotnet.md]:
```suggestion
![The pan button highlighted in the workflow designer.](./media/whats-new-in-wf-in-dotnet/pan-button-workflow-designer.png)
```

File: docs/framework/windows-workflow-foundation/whats-new-in-wf-in-dotnet.md [hunk 12]
```diff
@@ -149,7 +149,7 @@ In order to make hierarchical workflows easier to navigate, components of a work
 
 The following screenshot of the completed workflow from the [Getting Started Tutorial](getting-started-tutorial.md) shows the outline view with a sequential workflow.
 
-![Outline View in Workflow Designer](./media/outlineviewinworkflowdesigner.jpg "OutlineViewinWorkflowDesigner")
+![Outline view with a sequential workflow.](./media/whats-new-in-wf-in-dotnet/outline-view-in-workflow-designer.jpg)
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/whats-new-in-wf-in-dotnet.md]:
```suggestion
![Screenshot of outline view with a sequential workflow in Visual Studio.](./media/whats-new-in-wf-in-dotnet/outline-view-in-workflow-designer.jpg)
```

File: docs/framework/windows-workflow-foundation/whats-new-in-wf-in-dotnet.md [hunk 13]
```diff
@@ -227,7 +227,7 @@ State machine workflows were introduced as part of the .NET Framework 4, version
 
 The following screenshot shows the completed state machine workflow from the [Getting Started Tutorial](getting-started-tutorial.md) step [How to: Create a State Machine Workflow](how-to-create-a-state-machine-workflow.md).
 
-![Completed State Machine Workflow](./media/wfstatemachinegettingstartedtutorialcomplete.JPG "WFStateMachineGettingStartedTutorialComplete")
+![Completed state machine workflow from the Getting Started Tutorial](./media/whats-new-in-wf-in-dotnet/getting-started-tutorial-complete-state-machine-workflow.jpg)
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/whats-new-in-wf-in-dotnet.md]:
```suggestion
![Illustration that shows the completed state machine workflow.](./media/whats-new-in-wf-in-dotnet/complete-state-machine-workflow.jpg)
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @damabe. A few changes before we merge this one.

**Comment by damabe** [comment]
@mairaw Feedback incorporated. Updated CATS report:

https://contentqacats.azurewebsites.net/CheckResults/2019051500065/Report/

**Review by mairaw — APPROVED** [review]

**Pull request merged**